include/SFML/Audio.hpp

```diff
@@ -43,7 +43,9 @@
 #include <SFML/Audio/SoundRecorder.hpp>
 #include <SFML/Audio/SoundSource.hpp>
 #include <SFML/Audio/SoundStream.hpp>
-
+#include <SFML/Audio/ReverbEffect.hpp>
```

Comment:
I think effect headers should be separated via blank newline and sorted alphabetically here